doc/topics/windows/windows-package-manager.rst

```diff
@@ -853,25 +849,16 @@ example:
         uninstall_flags: '/qn /norestart'
         msiexec: True
 
-The above example shows us two things. First it demonstrates the usage of
-``latest`` as the version. In this case Salt will install the version of Chrome
-at the URL and report that version.
-
-The second thing to note is that this is installing software using an MSI. You
-can see that ``msiexec`` is set to ``True``.
+In the above example,
+- ``Version`` is set to ``latest``. Salt then installs the latest version of Chrome at the URL and displays that version.
+- ``msiexec`` is set to ``True`` hence the software is installed using an MSI.
 
-Example: MSI Patch
-==================
+Example: Package definition file to install an MSI patch
+=========================================================
+For MSI installer, when the ``msiexec`` parameter is set to true ``/i`` option is used for installation,
+and ``/x`` option is used for uninstallation. However, to install an MSI patch ``/i`` and ``/x`` options cannot be combined.
 
-When the ``msiexec`` parameter is set to ``True`` it uses the ``/i`` option for
-installs and the ``/x`` option for uninstalls. This is problematic when trying
-to install an MSI patch which requires the ``/p`` option. You can't combine the
-``/i`` and ``/p`` options. So how do you apply a patch to installed software in
-winrepo using an ``.msp`` file?
-
-One wiley contributor came up with the following solution to this problem by
-using the ``%cd%`` environment variable. Consider the following software
-definition file:
+Here is an example of a package definition file to install an MSI patch
```

Comment:
**Nitpick (grammar, blocking):** Add a colon (:) at the end of this sentence.

Reply:
Updated